src/k8s-engine/wsl.ts

```diff
@@ -1489,6 +1493,7 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     this.currentAction = Action.STOPPING;
     try {
       this.setState(K8s.State.STOPPING);
+      this.vtun.stop();
```

Comment:
`stop` is async. Either `await this.vtun.stop();`, or `this.vtun.stop().catch(e => console.error(e));`.